src/main/java/org/icgc/argo/program_service/services/ego/EgoService.java

```diff
@@ -273,6 +277,10 @@ public void initAdmin(String adminEmail, String shortName) {
   public List<User> getUsersInProgram(String programShortName) {
     val userResults = new ArrayList<User>();
     for (val role : roles()) {
```

Comment:
Just thinking if this validation should be added on method `roles()` instead?

Reply:
As theyre are parts of the application rely on roles() to return all roles, excluding "DEFAULT" globally could lead to unintended consequences.